shared/src/commonMain/kotlin/com/innova/innovaPoolingLibrary/DateTimeFormatter.kt

```diff
@@ -0,0 +1,13 @@
+package com.innova.innovaPoolingLibrary
+
+val dateUtilities = DateUtilities()
+
+class DateUtilities {
+    var platformDate = DateFormatter()
+    val outputDateFormat = "MMM d"
```

Comment:
outputDateFormat must be passed  into the function (getFormattedDate) parameter from platform side